pages/[...slug].vue

```diff
@@ -0,0 +1,130 @@
+<script setup>
+// COMPONENT RE-IMPORTS
+// TODO: remove when we have implemented component library as a module
+// https://nuxt.com/docs/guide/directory-structure/components#library-authors
+import {
+  TwoColLayoutWStickySideBar, NavBreadcrumb, ResponsiveImage, CardMeta, RichText, PageAnchor, FlexibleBlocks, SectionWrapper,
+} from 'ucla-library-website-components'
+
+import { onMounted } from 'vue'
+
+// HELPERS
+import _get from 'lodash/get'
+
+// GQL
+import FTVA_GENERAL_CONTENT_DETAIL from '../gql/queries/FTVAGeneralContentDetail.gql'
+
+// UTILS
+import getHeaders from '~/utils/getHeaders'
+
+const { $graphql } = useNuxtApp()
+
+const route = useRoute()
+
+const path = route.path.replace(/^\/|\/$/g, '') // trim initial and/or final slashes
+
+// Because we gencontent page uses ftva / in the uri to differentiate between the lib and meat websites the GQL query will need the slug instead of the uri
```

Comment:
this comment needs to be fixed

Reply:
```
// Because the generalcontent page uses ftva / in the uri
// to differentiate between the library and meap websites
// the GQL query will need the slug instead of the uri
```